config/schema/og.schema.yml

```diff
@@ -110,6 +110,9 @@ og.og_role.*:
     group_bundle:
       type: string
       label: 'Group bundle'
+    is_admin:
```

Comment:
@amitaibu Yes, that is what I'm shifting here. Hmm, wouldn't we need to do that anyway? As the permissions generated for OG could E.g. apply to one group, and therefore roles have to be check per group like that? The permissions for OG are kind of sub permissions I thought (OgRole !== global administer groups permission)? So this utilises all the isAdmin() stuff that is already a part of core Roles. So Core is already doing this with it's roles. I.e. if it's marked as a full administrator role, you can do anything. That same assumption applies on the OG level. E.g. a user membership has a role attached that is an admin role (is_admin = true) and they can then be a group admin. So all checks in OgAccess with check that anyway. E.g Role::hasPermission() (used by OgRole) will also check if the role isAdmin() first.